backend/src/auth/RIGHTS.ts

```diff
@@ -18,6 +18,8 @@ export enum RIGHTS {
   SET_PASSWORD = 'SET_PASSWORD',
   UPDATE_USER_INFOS = 'UPDATE_USER_INFOS',
   HAS_ELOPAGE = 'HAS_ELOPAGE',
+  CREATE_TRANSACTION_LINK = 'CREATE_TRANSACTION_LINK',
+  QUERY_TRANSACTION_LINK = 'QUERY_TRANSACTION_LINK',
```

Comment:
Do you assign this properly in inalienable rights?

Reply:
This must be set as an inalienable right